thecourseforum/theCourseForum
# Make /browse a react component

# What's changed
`/browse` is now 99% a react component! 

# How to test
1. Ensure that app is running via `docker-compose up`
2. Navigate to `localhost:3000/browse'
3. Ensure that links are working as expected!

# TODO
- fix testing to handle react code

# Screenshot
Looks nearly identical to before (as it should be) except for some slight padding differences
![image](https://user-images.githubusercontent.com/10587608/56336495-b0adfc80-616e-11e9-8d61-2bf1a871d373.png)
